src/pybind/mgr/dashboard/ci/check_grafana_dashboards.py

```diff
@@ -76,6 +76,22 @@ def get_grafana_dashboards(base_dir):
             with open(json_file) as f:
                 dashboard_config = json.load(f)
                 uid = dashboard_config.get('uid')
+                # if it's not a grafana dashboard, skip checks
+                # Fields in a dasbhoard:
+                # https://grafana.com/docs/grafana/latest/dashboards/json-model/#json-fields
+                expected_fields = [
+                    'id', 'uid', 'title', 'tags', 'style', 'timezone', 'editable',
+                    'hideControls', 'graphTooltip', 'panels', 'time', 'timepicker',
+                    'templating', 'annotations', 'refresh', 'schemaVersion', 'version', 'links',
+                ]
```

Comment:
Nice! I'd have assumed that grafonnet would set all these fields... Any clue why that's not happening?

Reply:
I think that ceph-cluster.json is not created in grafonnet. Is this correct @aaSharma14 ? @epuertat

Reply:
Yes, ceph-cluster.json is not created in grafonnet